src/main/java/no/nav/veilarbperson/client/pdl/PersonV2Data.java

```diff
@@ -40,11 +40,17 @@
     Oppholdsadresse oppholdsadresse;
     List<Kontaktadresse> kontaktadresser;
     String malform;
+    List<HentPdlPerson.Fullmakt> fullmakt;
+    List<HentPdlPerson.VergemaalEllerFremtidsfullmakt> vergemaalEllerFremtidsfullmakt;
+    boolean harVergemaal;
```

Comment:
Trenger ikke felt for om man har vergemål eller fullmakt. Det kan de som bruker endepunktet finne ut ved å sjekke på 
`
List<HentPdlPerson.Fullmakt> fullmakt;
 List<HentPdlPerson.VergemaalEllerFremtidsfullmakt> vergemaalEllerFremtidsfullmakt;
`